fixtures/command_line_tool_with_dynamic_dependencies/Project.swift

```diff
@@ -8,11 +8,17 @@ let project = Project(name: "CommandLineTool",
                                bundleId: "com.example.commandlinetool",
                                infoPlist: .default,
                                dependencies: [
+                                .target(name: "DynamicFramework"),
                                 .target(name: "DynamicLib"),
                                ]),
-                        Target(name: "DynamicLib",
+                        Target(name: "DynamicFramework",
                                platform: .macOS,
                                product: .framework,
+                               bundleId: "com.example.dynamicframework",
+                               infoPlist: .default),
+                        Target(name: "DynamicLib",
+                               platform: .macOS,
+                               product: .dynamicLibrary,
```

Comment:
thanks for changing the fixture!

I believe you'll have to create a new fixture, though (I'd keep the name of this the same and create a new one called `command_line_tool_with_framework_dependencies`) - if you ran the current fixture *without* your code changes, the test would still pass as the fixture asserts `Then tuist lints the project and fails` which in this case still does because of that `.framework` dependency.

Let me know if you need any help with writing the fixture 🙂

Reply:
✅